src/http/client/response.cr

```diff
@@ -1,154 +1,156 @@
 require "../common"
 require "mime/media_type"
 
-class HTTP::Client::Response
-  getter version : String
-  getter status : HTTP::Status
-  getter status_message : String?
-  getter headers : Headers
-  getter! body_io : IO
-  @cookies : Cookies?
-
-  def initialize(@status : HTTP::Status, @body : String? = nil, @headers : Headers = Headers.new, status_message = nil, @version = "HTTP/1.1", @body_io = nil)
-    @status_message = status_message || @status.description
-
-    if Response.mandatory_body?(@status)
-      @body = "" unless @body || @body_io
-    else
-      if (@body || @body_io) && (headers["Content-Length"]? != "0")
-        raise ArgumentError.new("Status #{status.code} should not have a body")
+class HTTP::Client
+  class Response
```

Comment:
Why this change? It seems completely unrelated.

Reply:
It is necessary because `win32_std_spec.cr` now requires this file from `http/common.cr`, and without this `HTTP::Client` would be first declared as a module which breaks further requires.

Reply:
By the way, similar namespaced types under `HTTP::Server` do this already.